crates/agglayer-storage/src/columns/proof_per_certificate/tests.rs

```
@@ -32,3 +36,43 @@ fn can_parse_value() {
 
     assert!(matches!(expected_value, Proof::SP1(_)));
 }
+
+#[derive(Deserialize)]
+struct SP1ProofWithPublicValuesV3 {
+    pub proof: sp1_sdk::SP1Proof,
+    #[allow(unused)]
+    pub stdin: sp1_sdk::SP1Stdin,
+    pub public_values: sp1_sdk::SP1PublicValues,
+    pub sp1_version: String,
+}
+
+impl SP1ProofWithPublicValuesV3 {
+    fn load(path: impl AsRef<std::path::Path>) -> Result<Self, Box<dyn std::error::Error>> {
```

Would it make sense to replace with `anyhow::Result`?

---

I don't think it is valuable here as the error will be ignored in the test